bitbots_dynamic_kick/cfg/bitbots_dynamic_kick_params.cfg

```diff
@@ -13,20 +13,20 @@ group_engine = gen.add_group("Engine", type="tab")
 group_engine_distances = group_engine.add_group("Distances and Positions")
 group_engine.add("engine_rate", int_t, 1,
                  "How often the engine updates motor goals [Hz]",
-                 500, min=1, max=500)
+                 0, min=1, max=500)
```

Comment:
there should be no default values in the .cfg but only in the config files. This can lead to issues. just specify the min and max value here